pyexchange/exchange2010/soap_request.py

```diff
@@ -7,6 +7,10 @@
 from lxml.builder import ElementMaker
 from ..utils import convert_datetime_to_utc
 
+import sys
+if sys.version_info[0] == 3:
+	unicode = str
```

Comment:
The doc states "PyExchange supports Python 2.6 and 2.7. **Our code is compatible with 3.3+**, but see the notes below on getting it working."
Besides the fact that the "notes" on ntlm are not valid anymore (``pip3 install pyexchange`` now works out of the box AFAIK), running the [this code](https://pyexchange.readthedocs.org/en/latest/#listing-events) from the docs under python3 will fail with ``NameError: name 'unicode' is not defined`` or similar error.
Maybe this would be more visible if python3 was enabled on Travis C.I./coverage ?

The culprit is in get_calendar_item around line 130
```python
       u'MaxEntriesReturned': unicode(max_entries),
```
BTW, same issue in exchange2010/__init__.py in move_to with:
```python
    if not isinstance(folder_id, basestring):
```
I will fix that one too, probably along with some mail support :-)
Do you prefer separate PRs ?